### ISSUE TITLE
remove duplicate import from _canny.py

### ISSUE THREAD

**Issue by judafe** [scikit-image/scikit-image#6455]
### Problem
[_canny.py](https://github.com/scikit-image/scikit-image/blob/main/skimage/feature/_canny.py#L12) imports ```gaussian``` twice. See lines 12 and 14.

```python
from .._shared.filters import gaussian # first import (line 12)
from ..filters._gaussian import gaussian # second import  (line 14)
```
The import from [_gaussian.py](https://github.com/scikit-image/scikit-image/blob/38b595d60befe3a0b4c0742995b9737200a079c6/skimage/filters/_gaussian.py) simply points to the ```gaussian``` function in [filters.py.](https://github.com/scikit-image/scikit-image/blob/38b595d60befe3a0b4c0742995b9737200a079c6/skimage/_shared/filters.py). The top of _gaussian.py is pasted here for convinience...

```python
from collections.abc import Iterable

import numpy as np
from scipy import ndimage as ndi

from .._shared import utils
from .._shared.filters import gaussian
from .._shared.utils import _supported_float_type, convert_to_float, warn
from ..util import img_as_float

__all__ = ['gaussian', 'difference_of_gaussians']
```

### Solution
**Remove duplicate.** I have already completed this and will be pushing the changes to my fork. Please let me know if there is a reason to maintain the duplicate imports. 

## Way to reproduce
Not applicable

## Version information
Not applicable

**Comment by rfezzani** [comment]
Good catch @judafe, you can go ahead and make a PR, Thank you ;)

**Comment by rfezzani** [comment]
Fixed by #6457
